packages/connect-voting-disputable/src/thegraph/parsers/settings.ts

```diff
@@ -0,0 +1,64 @@
+import { QueryResult } from '@aragon/connect-thegraph'
+
+import Setting from '../../models/Setting'
+import { SettingData } from '../../types'
+
+function buildSetting(setting: any, connector: any): Setting {
+  const {
```

Comment:
I wonder if we may use spread syntax to build the `settingData` we need?

Reply:
not sure I follow :)

Reply:
I mean something like:
```
const settingData = {
    ...setting,
    votingId: setting.voting.id
}
```

But maybe if we do that we break the `SettingData` interface 🤔

Reply:
Yes, it breaks it cause we will include some objects like `setting.voting` that are not part of the interface, I think at some point we need to do some filtering. 

What we could do is to use `Object.assign` in the entity constructor since it is receiving an object that implements the expected interface